<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherDialog</class>
 <widget class="QDialog" name="TeacherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>791</width>
     <height>601</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>194</red>
         <green>194</green>
         <blue>194</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>126</red>
         <green>126</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>194</red>
         <green>194</green>
         <blue>194</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>126</red>
         <green>126</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>126</red>
         <green>126</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>126</red>
         <green>126</green>
         <blue>126</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="ProfileTab">
    <attribute name="title">
     <string>Профиль</string>
    </attribute>
    <widget class="QLabel" name="studentNameLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ФИО препа</string>
     </property>
    </widget>
    <widget class="QLabel" name="studentGroupLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ФИО</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editProfileButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Редактировать профиль</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Группы в подчинении</string>
     </property>
    </widget>
    <widget class="QTableView" name="labsTable_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>151</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Дать задание</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Группы</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Коллективы</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>0</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Члены коллектива/Лабы</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QListView" name="allGroupsListView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>161</width>
       <height>501</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>649</x>
       <y>530</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Назначить</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>601</width>
       <height>501</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListView" name="collectivesListView">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QListView" name="studentsListView">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QListView" name="labsListView">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Добавить преподавателя</string>
    </attribute>
    <widget class="QLabel" name="nameLable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ФИО</string>
     </property>
    </widget>
    <widget class="QLabel" name="loginLable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameLine">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="loginLine">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLine">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="password2Lable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Повторить пароль</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password2Line">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="subjectsLable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Предметы</string>
     </property>
    </widget>
    <widget class="QLabel" name="groupsLable">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>210</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Группы</string>
     </property>
    </widget>
    <widget class="QPushButton" name="singUpButton">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>500</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Зарегистрировать</string>
     </property>
    </widget>
    <widget class="QListView" name="subjectsListView">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>231</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QListView" name="groupsListView">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>210</y>
       <width>221</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="LabsTab">
    <attribute name="title">
     <string>Лабораторные</string>
    </attribute>
    <widget class="QLabel" name="labsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>17</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Лабы</string>
     </property>
    </widget>
    <widget class="QTableView" name="labsTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>57</y>
       <width>381</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="newlabButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dellabButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>110</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QLabel" name="tasksLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>317</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Задание</string>
     </property>
    </widget>
    <widget class="QTableView" name="tasksTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>357</y>
       <width>380</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="descriptionLabel">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>317</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Описание</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="descriptionLine">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>357</y>
       <width>321</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
